### ISSUE TITLE
A11y_Model Builder_Create New Experiment_Usability: 'Create New Experiment' is not defined as heading. Narrator is not announcing heading in the popup

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [dotnet/machinelearning-modelbuilder#1577]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**
#A11y_Model Builder; #A11yUsable; #A11yTCS; #Win32; #DesktopApp; #A11ySev4; #ModelBuilder-Win32-Jun2021; #Visual Studio Client;

### Environment Details:
Application name: Model Builder

### OS Details:
Windows Version 21H1 OS Build (19043.1083)
Microsoft Visual Studio 

### Note:
This issue is observed in all popups. 'Create new workspace', 'Create new resource groups', 'Create new compute' are also not defined as headings

### Repro Steps:

1. Turn on Narrator.
2. Enable ML.NET Model builder as a preview feature in tools>options>Environment>preview features.
2. Create a .NET C# project and tab till project under solution explorer and Add machine learning.
3. Tab till Image classification under scenario tab and press enter.
4. Select Azure under environment tab and tab till set up workspace and press enter.
5. In 'Create New Experiment' dialog, turn on scan mode using Capslock + space keys, and press h.
6. Verify that Narrator announces as no previous/next headings in the popup

### Actual Result:
'Create New Experiment' is not defined as heading. Narrator is not announcing heading in the popup

### Expected Result:
'Create New Experiment' should be defined as heading. Narrator should announce headings in the popup in scan mode. In all popups, all the headings should be defined in programmatic way. 'Create new workspace', 'Create new resource groups', 'Create new compute' should also be defined as headings.

### User Impact:
End users will not clearly understand the purpose of the popup if heading is not defined in programmatic way.

**Comment by beccamc** [comment]
Validate on rel/candidate build 16.7.2.2141903. See issue 1627 for instructions for install.

**Comment by beccamc** [comment]
Here is a link the most recent Model Builder build. I’ve tested it on VS 17.0.0 Preview 5 

Model Builder: https://devdiv.visualstudio.com/_apis/resources/Containers/8681620/VSSetup_Release17?itemPath=VSSetup_Release17%2FMicrosoft.ML.ModelBuilder.Gallery.Dev17.vsix
VS: https://aka.ms/vs/17/pre/vs_Enterprise.exe

**Comment by NiharikaTipirishetty** [comment]
@beccamc Issue is not fixed. Narrator is still not reading headings in the popup. Since this is a usability bug there is no compliance blocking, please add this bug to your backlog, If you are not going to fix this issue, you can feel free to close this bug

![1577_not fixed](https://user-images.githubusercontent.com/83570638/134012229-b9ae92ea-cbfe-4af2-998e-a3bc831381a4.png)

**Comment by beccamc** [comment]
Thanks Niharika! I'll mark this as resolved for the accessibility pass. @zewditu Can you look at why this isn't fixed?

**Comment by zewditu** [comment]
From my end on the latest Model Builder, it announces "Asure machine learning. create new experiment window "

**Comment by zewditu** [comment]
The narrator announces the header in scan mode too.
